### PR TITLE
Improve readme and badges

### PR DESCRIPTION
**Pull request by jrmi**
To be merged after the previous one.

### CODE REVIEW

File: pypeman/nodes.py [hunk 1]
```diff
@@ -247,8 +247,15 @@ def process(self, msg):
 
 class Drop(BaseNode):
```
Comment (feenes) [inline pypeman/nodes.py]: English:
This node can be used
This node is used
Use this node

"this node used to" means that before the node had the mentioned functionality, but it has it no more.

File: tox.ini [hunk 2]
```diff
@@ -2,26 +2,26 @@
...
-    nose
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/requirements_test.txt
-commands=python -m setup test
-passenv=PYPEMAN_TEST_SMTP_USER PYPEMAN_TEST_SMTP_PASSWORD PYPEMAN_TEST_RECIPIENT_EMAIL
-
 [testenv:py35]
 deps =
     nose
+    coverage
```
Comment (feenes) [inline tox.ini]: why add coverage and codecov here. shouldn't 
`r{toxinidir}/requirements_test.txt`
fetch it already?

File: pypeman/nodes.py (hunk 1 above)
Comment (jrmi) [reply pypeman/nodes.py]: Fixed

File: tox.ini (hunk 2 above)
Comment (jrmi) [reply tox.ini]: Don't remember why but seems working without that...

**Pull request merged**